packages/web-react/src/utils/string.ts

```diff
@@ -1,6 +1,27 @@
 /**
- * Converts a kebab-case string to camelCase
+ * Converts a kebab-case string or an object with kebab-case string values to camelCase.
```

Comment:
**suggestion (if-minor):** I think that now it does 2 things: converting a string and parsing an object.

To keep this simple I suggest creating a new function that will be able to parse the object and call the kebabCaseToCamelCase on its strings/values.